### ISSUE TITLE
Support Firefox?

### ISSUE THREAD

**Issue by Pomax** [amokjs/amok#10]
How do I run this with Firefox's remote debugger instead of Chrome?

**Comment by caspervonb** [comment]
Currently only chrome is supported, different protocols and the firefox one is is a "little" bit harder to understand. Hopefully this will change in the not so distant future.

**Comment by Pomax** [comment]
ah. I'll be on the lookout for when you do. Until then I guess I'll stick with live-reload/webpack (mostly because chrome-only means it can't be used for automated testing).

**Comment by caspervonb** [comment]
> means it can't be used for automated testing

Could you elaborate on this use case? :smile:

**Comment by Pomax** [comment]
certainly. For our devs it's important to test all the code they're writing in FF and Chrome at the very least, so one thing we do is run our entire dev cycle through an automated `gulp` task - this constantly applies jsling and jscs, and makes sure all assets are synced, and usually we have that open in both browsers, which right now autoupdate for CSS changes but not JS. Using amok, we could have this also do the live JS loading, but as is that means it'll autoupdate in Chrome, but requires manual reloading in Firefox, which is arguably worse than knowing to just "reload all browsers" =)

**Comment by Pomax** [comment]
closed because "won't ever do", "already done", or "there is another ticket for this"?

**Comment by caspervonb** [comment]
For now, it's not possible.

**Comment by caspervonb** [comment]
Pending on this issue https://bugzilla.mozilla.org/show_bug.cgi?id=1151500

**Comment by caspervonb** [comment]
Just to clarify intent, re-opening with with a new label, still working out kinks of issue management ;)

**Comment by Pomax** [comment]
nice, I left a comment on the bugzilla issue against Firefox about it.

**Comment by stereokai** [comment]
@Pomax Have you taken a look at [BrowserSync](https://github.com/BrowserSync/browser-sync)?

**Comment by caspervonb** [comment]
Not actionable right now, nor in the near future, closing for the time being.
